internal/pkg/flink/internal/controller/interactive_output_controller_test.go

```diff
@@ -243,3 +241,99 @@ func (s *InteractiveOutputControllerTestSuite) TestTableTitleDisplaysPageSizeAnd
 
 	cupaloy.SnapshotT(s.T(), actual)
 }
+
+func (s *InteractiveOutputControllerTestSuite) TestArrowUpOrDownStopsFetchWhenAutoRefreshIsRunning() {
```

Comment:
Maybe `TestArrowUpOrDownToggleAutoRefreshWhenAutoRefreshIsRunning` would be a bit more precise, right? We don't/can't really check if it stops.

Reply:
Renamed it!